index.html

```diff
@@ -641,8 +641,9 @@ <h2>Dereferencing the Secondary Resource</h2>
 				<li>From the <var>resolved <a>DID document</a></var>, select the JSON-LD object whose <code>id</code>
 				property matches the <var>input <a>DID URL</a></var>, e.g. a public key or service endpoint in the
 				<a>DID document</a>. This is called the <var>output resource</var>.
-				Selecting the JSON-LD object from the <a>DID document</a> MUST take into account relative IRIs and
-				the base IRI of the <var>resolved <a>DID document</a></var>.</li>
+				When selecting the JSON-LD object from the <a>DID document</a>, the absolute
+				<a>DID URL</a> used to identify a graph node MUST be unique and present only once in the
+				<a>DID document</a>. If the indentifer of the graph node is not unique, an error MUST be thrown.
```

Comment:
```suggestion
				property matches the <var>input <a>DID URL</a></var>, e.g., a public key or service endpoint in the
				<a>DID document</a>. This is called the <var>output resource</var>.

				When selecting the JSON-LD object from the <a>DID document</a>, the absolute
				<a>DID URL</a> used to identify a graph node MUST be unique and present only once in the
				<a>DID document</a>. If the identifier of the graph node is not unique, an error MUST be thrown.
```

Reply:
I'm not able to figure out what this suggestion is doing. The only difference I see here is that line 644 now has a space between the two. Given these are in a `<li>` HTML format I'm not sure what this adds. Can you clarify the suggestion @TallTed ?